src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -207,6 +207,15 @@ Resources:
       PolicyDocument:
         Version: "2012-10-17"
         Statement:
+          - Effect: Allow
```

Comment:
I think we should only add this statement when Terraform mode is enabled in the ADF.
Otherwise all customers that do not rely on Terraform would get DynamoDB tables created and permissions added that don't add any value to them.

Reply:
Added a parameter in adfconfig.yml for Terraform extensions and a condition to deploy DynamoDB table only if the parameter is set to True